### PR TITLE
Empty space in Mobile view in Event Detail Page

### LINKED ISSUE #2694: Empty space in Event Details Page after License Info
**Describe the bug**
An empty space is present in Mobile view in the Event Detail Page.

**Expected behaviour**
Empty space should not be there.

**Screenshots**
Screenshot from 2019-04-16 21-02-16

**Additional context**

Working on it.

### PR DESCRIPTION
**Pull request by prateekj117**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Empty space at the end of the event detail page in mobile view. 

Fixes #2694

### CODE REVIEW

**Comment by prateekj117** [comment]
@CosmicCoder96 @ritikamotwani Please review.

File: app/templates/public.hbs [hunk 1]
```diff
@@ -38,11 +38,17 @@
       <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="three wide column public">
```
Comment (uds5501) [inline app/templates/public.hbs]: @prateekj117 Please see if your issue is solved by replacing this line with :

```hbs
<div class="{{#if device.isMobile 'three wide column public' ''}}
```

Comment (CosmicCoder96) [reply app/templates/public.hbs]: @prateekj117 make this change.

**Comment by prateekj117** [comment]
@CosmicCoder96 Done, please review now.

File: app/templates/public.hbs [hunk 2]
```diff
@@ -38,7 +38,7 @@
...
+      <div class="{{if device.isMobile '' 'three wide column public'}}">
```
Comment (CosmicCoder96) [inline app/templates/public.hbs]: Don't leave the class empty for a mobile screen, divs with no classes are a bad practice.

**Comment by prateekj117** [comment]
@CosmicCoder96 Please review now.

File: app/templates/public.hbs [hunk 3]
```diff
@@ -38,7 +38,7 @@
       <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="three wide column public">
+      <div class="{{if device.isMobile 'external-link' 'three wide column public'}}">
```
Comment (CosmicCoder96) [inline app/templates/public.hbs]: what's `external-link` ? by not leaving it empty, I meant if it's a three-wide column on a large screen, then there is a column size which is correct for mobile screen. Use those semantic UI classes.

**Comment by prateekj117** [comment]
@CosmicCoder96 Please review now.

File: app/templates/public.hbs [hunk 4]
```diff
@@ -38,7 +38,7 @@
       <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="three wide column public">
+      <div class="{{if device.isMobile 'three wide column' 'three wide column public'}}">
```
Comment (CosmicCoder96) [inline app/templates/public.hbs]: @prateekj117 It can be optimized to 
```hbs
      <div class="three wide column {{if device.isMobile 'public'}}">
```

**Comment by prateekj117** [comment]
@CosmicCoder96 Done. Please review.

**Pull request merged**